### PR TITLE
config: fix `binlog-gtid` verify

### LINKED ISSUE #978: "binlog-name must specify" reported when `binlog-gtid` specified
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

   `start-task` with `binlog-gtid` (without `binlog-name` specified, and `enable-gtid: true` in the source config)

2. What did you expect to see?

   the task works well.

3. What did you see instead?

    `[code=20011:class=dm-master:scope=internal:level=medium], Message: mysql-instance: 0th: source mysql-replica-gop-02: binlog-name must specify, Workaround: Please check the `meta` config in task configuration file.`

4. Versions of the cluster

    - DM version (run `dmctl -V` or `dm-worker -V` or `dm-master -V`):

        ```
        v2.0.0-rc
        ```

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

fix #978 

### What is changed and how it works?

   check `binlog-gtid` in `Verify`.

### Check List 

Tests 

 - Unit test
 - Integration test

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by lance6716** [comment]
`make terror`. but seems test check of CI printing usage message 🤔 

https://internal.pingcap.net/idc-jenkins/blue/rest/organizations/jenkins/pipelines/dm_ghpr_test/runs/6540/nodes/93/steps/183/log/?start=0

File: dm/config/task_test.go [hunk 1]
```diff
@@ -555,3 +555,38 @@ func (t *testConfig) TestFromSubTaskConfigs(c *C) {
 
 	c.Assert(cfg.String(), Equals, cfg2.String()) // some nil/(null value) compare may not equal, so use YAML format to compare.
 }
+
+func (t *testConfig) TestMetaVerify(c *C) {
+	var m *Meta
+	c.Assert(m.Verify(), IsNil) // nit meta is fine (for not incremental task mode)
```
Comment (lance6716) [inline dm/config/task_test.go:561]:
```suggestion
	c.Assert(m.Verify(), IsNil) // nil meta is fine (for not incremental task mode)
```

**Review by lance6716 — COMMENTED** [review]
not sure how this PR solve [origin issue](https://github.com/pingcap/dm/issues/978)

**Comment by csuzhangxc** [comment]
> not sure how this PR solve [origin issue](https://github.com/pingcap/dm/issues/978)

I tiny updated the origin issue.

previous, specify `binlog-gtid` without `binlog-name` will return the error; now, the user can only specify `binlog-gtid` (without `binlog-name`).

**Comment by lance6716** [comment]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**